eng/ci/templates/official/jobs/run-benchmarks.yml

```diff
@@ -16,10 +16,12 @@ jobs:
   variables:
     runDescription: ${{ parameters.description }}
     functionApp: ${{ parameters.functionAppName }}
-    functionAppOutputPath: $(Build.ArtifactStagingDirectory)\FunctionApps\$(functionApp)
-    benchmarkResultsJsonPath: "$(Build.ArtifactStagingDirectory)\\BenchmarkResults\\$(functionApp).json"
+    functionAppOutputPath: $(Build.ArtifactStagingDirectory)/FunctionApps/$(functionApp)
+    benchmarkResultsJsonPath: "$(Build.ArtifactStagingDirectory)/BenchmarkResults/$(buildNumber)_$(functionApp).json"
```

Comment:
Curious what the need for `$(buildNumber)_` here is?

Reply:
I added it for readability of the file name (if one were to download the artifacts from multiple builds they queue from their branch)